### PR TITLE
feat: add upgrade to Terraform 0.13.7

### PR DESCRIPTION
**Pull request by blgm**
[#181155156](https://www.pivotaltracker.com/story/show/181155156)

### Checklist:

* [x] Have you added Draft Release Notes in `docs/draft-release-notes.md`?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

File: terraform/postgresql/bind/versions.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    postgresql = {
+      source  = "cyrilgdn/postgresql"
+      version = ">= 1.16.0"
+    }
+    csbpg = {
+      source  = "cloud-service-broker/csbpg"
+      version = ">= 1.0.0"
```
Comment (jhvhs) [inline terraform/postgresql/bind/versions.tf:9]: This version should be >= 1.0.1

File: terraform/postgresql/bind/versions.tf [hunk 2]
```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_providers {
+    postgresql = {
```
Comment (FelisiaM) [inline terraform/postgresql/bind/versions.tf:3]: Are we still using this?

Comment (blgm) [reply terraform/postgresql/bind/versions.tf:3]: Yes, it's needed for the transition to the new provider.

File: terraform/postgresql/bind/versions.tf (hunk 1 above)
Comment (blgm) [reply terraform/postgresql/bind/versions.tf]: I've updated it.

**Pull request merged**